cloudinary/cloudinary_wordpress
# Fixes WPBakery insert error

Test instructions:

- Install WPBakery
- Insert Cloudinary images in a post using WPBakery editor
- Observe console and notice absence of errors
